### PR TITLE
css-two-column-layout | Richard Watanabe

### PR DESCRIPTION
**Pull request by Richard-Watanabe**
Screen Shot 2021-06-25 at 3 08 38 PM

### CODE REVIEW

File: css-two-column-layout/styles.css [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+  width: 30%;
+  position: relative;
+  top: 2.5rem;
+}
+
+button {
+  border-color: rgb(255, 47, 0);
+  color: rgb(255, 47, 0);
+  width: 8 rem;
+  height: 1 rem;
```
Comment (codingcodymiller) [inline css-two-column-layout/styles.css:72]: There should not be spaces between a value and its unit

File: css-two-column-layout/index.html [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+<body>
+  <div class="container">
+    <div class="row">
+      <div class="column-full">
+        <h1>Scale, Speed, Agility</h1>
+          <p>
+            We've dedicated thousands of man hours to making healthcare
+            institutions in the Helium network run 10x more efficiently
+            and deliver an unparalleled quality of care.
+          </p>
```
Comment (codingcodymiller) [inline css-two-column-layout/index.html:21]: Your indentation here is a bit off. Remember, sibling element should always be at the same level of indentation.

File: css-two-column-layout/index.html [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+    </div>
+    <div class="row">
+      <div class="column-half">
+        <img src="images/software.svg" alt="software-icon">
+          <h3>Intuitive, <br> Smart Software</h3>
+            <p class="body">
+              We build products that transform how healthcare facilities
+              run across Africa - making them radically more efficient
+              operationally and financially, a complete turnaround.
+            </p>
```
Comment (codingcodymiller) [inline css-two-column-layout/index.html:32]: Your indentation here is a bit off. Remember, sibling element should always be at the same level of indentation.

File: css-two-column-layout/index.html [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+      </div>
+      <div class="column-half">
+        <img src="images/practice.svg" alt="medical-icon">
+          <h3>Practice Medicine, Not Administration</h3>
+            <p class="body">
+              Our all-in-one electronic health records (EHR), hospital
+              management (HMIS) &amp; billing system helps streamline your facility,
+              reduce waste and improve quality of care.
+            </p>
```
Comment (codingcodymiller) [inline css-two-column-layout/index.html:41]: Your indentation here is a bit off. Remember, sibling element should always be at the same level of indentation.

File: css-two-column-layout/index.html [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+    </div>
+    <div class="row">
+      <div class="column-half">
+        <img src="images/circumstance.svg" alt="circumstance-icon">
+          <h3>For Any Circumstance</h3>
+            <p class="body">
+              Our intuitive system is built around the peculiarities of your facility;
+              we support offline deployments, sync to the cloud and run on any device
+              24/7.
+            </p>
```
Comment (codingcodymiller) [inline css-two-column-layout/index.html:52]: Your indentation here is a bit off. Remember, sibling element should always be at the same level of indentation.

File: css-two-column-layout/index.html [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+      </div>
+      <div class="column-half">
+        <img src="images/data.svg" alt="data-icon">
+          <h3>Data That Transforms</h3>
+            <p class="body">
+              We understand your facility better than you do; we provide visually-appealing
+              real-time analytics, reports and trends <br> at your finger tips.
+            </p>
```
Comment (codingcodymiller) [inline css-two-column-layout/index.html:60]: Your indentation here is a bit off. Remember, sibling element should always be at the same level of indentation.

File: css-two-column-layout/index.html [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+      </div>
+    </div>
+    <div class="row">
+      <div class="column-full">
+        <div class="button">
+        <button>Learn more</button>
```
Comment (codingcodymiller) [inline css-two-column-layout/index.html:66]: This should be indented one more tab

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Great start on this!!! A few notes for you:

**Comment by Richard-Watanabe** [comment]
Screen Shot 2021-06-25 at 3 50 05 PM

**Review by codingcodymiller — APPROVED** [review]
Great job on this!!!! 👏 👏 👏

**Pull request merged**